### PR TITLE
Add SourceLink, embedded readme, implicit usings; refactor project properties

### PR DESCRIPTION
**Pull request by sungam3r**
I need to debug one of our services that uses graphql-client.

### CODE REVIEW

File: Directory.Build.props [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    <PackageReference Include="Roslynator.Analyzers" Version="4.1.1">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
+    <PackageReference Include="Roslynator.Formatting.Analyzers" Version="4.1.1">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
+  </ItemGroup>
+
```
Comment (Shane32) [inline Directory.Build.props:43]:
```suggestion
```

Unnecessary.  Just set `<EnableNETAnalyzers>true</EnableNETAnalyzers>` like we do in the server repo

**Review by Shane32 — CHANGES_REQUESTED** [review]

**Review by rose-a — CHANGES_REQUESTED** [review]
The build generates packages for *GraphQL.Server.Test* and *IntegrationTestServer* now.

Please disable package generation for those (now enabled because of `<IsPackable>true</IsPackable>` in *Directory.Build.props*)

**Comment by sungam3r** [comment]
Done. Forgot to set IsPackable to false.

File: Directory.Build.props (hunk 1 above)
Comment (sungam3r) [reply Directory.Build.props]: ok

**Pull request merged**